members/PAL000460.yaml

```diff
@@ -39,8 +39,8 @@ contact_form:
       value: $MESSAGE
       required: true
   - click_on:
-    - value: Submit
-      selector: '#ctl00_ContentPlaceHolder1_btnSubmitContact'
+    - selector: '#ctl00_ContentPlaceHolder1_chkAddToEmailList'
+    - selector: '#ctl00_ContentPlaceHolder1_btnSubmitContact'
```

Comment:
i dont think this second selector is needed

Reply:
@annanhan tried taking out 'selector: '#ctl00_ContentPlaceHolder1_btnSubmitContact'' and it wasn't submitting when testing it. Is there a different way I should handle the submit step?

Reply:
oh shoot, that's the submit button. i misread what you were trying to do.

for clarity, can you break up the click_on steps? One for the check box and one for the submit and give them meaningful "names" 

so something like

``` 
- click_on:
    - name: Remove from Email Checkbox 
    - selector: '#ctl00_ContentPlaceHolder1_chkAddToEmailList'
- click_on:
    - name: Submit Button
    - selector: '#ctl00_ContentPlaceHolder1_btnSubmitContact'
```